--- mutt-1.8.0/init.c.orig	2017-02-27 12:32:06.258283812 +0100
+++ mutt-1.8.0/init.c	2017-02-27 12:32:25.226215774 +0100
@@ -3936,7 +3936,7 @@
       continue;
     }
 
-    hash_insert(TagTransforms, tag, transform, 0);
+    hash_insert(TagTransforms, tag, transform);
   }
   return 0;
 }
@@ -3967,7 +3967,7 @@
       continue;
     }
 
-    hash_insert(TagFormats, format, tag, 0);
+    hash_insert(TagFormats, format, tag);
   }
   return 0;
 }
